web_dashboard_tile/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+# © 2010-2013 OpenERP s.a. (<http://openerp.com>).
```

Comment:
Replace the copyright character by 'Copyright'

Reply:
Contributing modules for the first time. We will make the necessary changes and update. Thanks @tarteo